_layouts/role.html

```diff
@@ -4,97 +4,130 @@
 class:
 ---
 {% comment %}Gather the matching content from pages, playbook, guide, and posts.{% endcomment %}
-{% assign playbook = site.playbook | where: "roles", page.title %}
-{% assign guide = site.guide | where: "roles", page.title %}
-{% assign personas = site.personas | where: "roles", page.title %}
-{% assign projects = site.projects | where: "roles", page.title %}
-{% assign posts = site.posts | where: "roles", page.title %}
-{% assign books = site.data.books.docs | where: "roles", page.title %}
-{% assign tools = site.data.tools.docs | where: "roles", page.title %}
+{% assign playbook = site.playbook | where: "roles", page.title | sort: "title" %}
+{% assign guide = site.guide | where: "roles", page.title | sort: "title" %}
+{% assign personas = site.personas | where: "roles", page.title | sort: "title" %}
+{% assign projects = site.projects | where: "roles", page.title | sort: "title" %}
+{% assign posts = site.posts | where: "roles", page.title | sort: "title" %}
+{% assign books = site.data.books.docs | where: "roles", page.title | sort: "title" %}
+{% assign tools = site.data.tools.docs | where: "roles", page.title | sort: "title" %}
 {% include breadcrumbs.html %}
+<!--<script src="{{ base.url | prepend: site.url }}/assets/js/accordion.js"></script>-->
+<script src="http://127.0.0.1:4000/assets/js/accordion.js"></script>
```

Comment:
@jaube05 I think this can be just `/assets/js/accordion.js` so you don't have to worry about the site URL.